src/resources/formats/html/ojs/ojs-bundle.js

```diff
@@ -54,6 +54,10 @@ export class OJSInABox {
     // so any variables that are still missing do cause "x is not defined"
     // errors.
     this.pendingGlobals = {};
+    // When true, the mechanism described in the `this.pendingGlobals` comment
+    // is used. When false, the result of accessing undefined variables is just
+    // "x is not defined". This should be considered private, only settable via
+    // constructor or `killPendingGlobals`.
     this.allowPendingGlobals = allowPendingGlobals;
```

Comment:
I'm ok keeping this as-is, but I'm also wondering if we should be using the convention that underscore-prefixed variables are intended to be private.

Reply:
I had the same instinct (also for the `globals` method).